docs/source/tutorials/linops/linear_operators_quickstart.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>finite-dimensional [operator / vector space] instead of "finite". Just finite has a (potentially) different meaning in both contexts, namely finite-rank and finite cardinality (see e.g. https://en.wikipedia.org/wiki/Examples_of_vector_spaces#Finite_vector_spaces).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>